i18n/en.json

```diff
@@ -4176,7 +4176,7 @@
   },
   {
     "id": "mfa.validate_token.authenticate.app_error",
-    "translation": "Error trying to authenticate MFA token"
```

Comment:
@sbishel Wondering if this can be simplified to "Error authenticating MFA token"

Reply:
That is the old wording, I matched the new wording to when the code is bad. 
```{
    "id": "api.user.check_user_mfa.bad_code.app_error",
    "translation": "Invalid MFA token."
  },```

Reply:
My bad - this is good to go then from my perspective.  Thanks for clarifying